packages/web-api/src/methods.ts

```diff
@@ -234,6 +237,8 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
         list:
           bindApiCall<AdminAppsRestrictedListArguments, AdminAppsRestrictedListResponse>(this, 'admin.apps.restricted.list'),
       },
+      // TODO: Use AdminAppsUninstallResponse when it's available
```

Comment:
`AdminAppsUninstallResponse` will be available once we merge https://github.com/slackapi/java-slack-sdk/pull/735

Reply:
This is done.